reqhandler/fields.go

```diff
@@ -0,0 +1,33 @@
+package reqhandler
+
+type FieldName string
```

Comment:
I think I'm confused, because something called "Headers" exists inside request handler with parts that are similar to the names of the constants in fields.go. Why did you set up the data structures in this way?